playbooks/openshift-imagestreamtags/update_imagestreamtags.yml

```diff
@@ -0,0 +1,7 @@
+---
+- import_playbook: ../init/main.yml
+  
+- name: Update image stream tag
+  hosts: all
+  roles:
+    - openshift_imagestreamtags
```

Comment:
We've tried to establish a pattern of only putting "entry point" playbooks in playbooks/foo/ and then putting actual implementation in playbooks/foo/private/. The expectation being that only other playbooks which have previously run init would call directly to /private/

So to do that here split lines 4-7 off into playbooks/openshift-imagestreams/private/update_imagestreamtags.yml and then add something like the following here.

```
- import_playbook: private/update_imagestreamtags.yml
```

Also, I think hosts should be `oo_first_master` which will ensure that this runs only on the first master. That's what we do with most things that just interact with the api.

Reply:
Maybe also this should live in the existing playbooks/openshift-master/ tree? We don't really have well defined patterns for how we've named top level playbook directories.

Reply:
OK, I'll push the first part in a bit.  On the where to live piece, yeah, I struggled on where to put this, ease of access, etc.

In the end, it didn't seem to fit with what I saw under openshift-master, but I fully admit I don't have enough experience / context in this space, and will ultimately defer if strong opinions emerge.